TUTORIAL.md

```diff
@@ -0,0 +1,587 @@
+  <div>
+    <h1>Hello, world!</h1>
+  </div>
+)
+```
+
+This is the simplest page possible in Blitz. To look at it, go back to your browser and go to http://localhost:3000. You should see your text appear! Try editing the `index.tsx` file, and make it your own! When you’re ready, move on to the next section.
+
+
+## Database setup
+Now, we’ll setup the database and create your first model.
+
+Open up `db/schema.prisma`. It’s a configuration file which our default database engine Prisma uses.
+
+By default, the apps is created with SQLite. If you’re new to databases, or you’re just interested in trying Blitz, this is the easiest choice. Not that when starting your first project, you may want to use a more scalable database like PostgreSQL, to avoid the pains of switching your database down the road.
+
+## Creating models
+
+Now we’ll define your models—essentially, your database layout, with additional metadata.
+
+In `schema.prisma`, we’ll create two models: `Question`, and `Choice`. A `Question` has a question and a publication date. A `Choice` has two fields: the text of the choice and a vote count. Each has an id, and each `Choice` is associated with a `Question`.
+
+Edit the `schema.prisma` file so it looks like this:
+
+```
+// (datasource and generator)
+
+...
+
+model Question {
+  id          Int      @default(autoincrement()) @id
+  text        String
+  publishedAt DateTime
+  choices     Choice[]
+}
+
+model Choice {
+  id         Int      @default(autoincrement()) @id
+  text       String
+  votes      Int      @default(0)
+  question   Question @relation(fields: [questionId], references: [id])
+  questionId Int
+}
+```
+
+Now, we need to migrate our database. This is a way of telling it that you have edited your schema in some way. Run this, and enter something like `init db` when prompted for a migration name:
+
+```sh
+$ blitz db migrate
+```
+
+
+## Playing with the API
+
+Now, let’s hop into the interactive Blitz shell and play around with the free API Blitz gives you. To invoke the Blitz shell, use this command:
+
+```sh
+$ blitz console
+```
+
+Once you’re in the console, explore the Database API:
+
+
+
+```sh
+# No questions are in the system yet.
+⚡ > await db.question.findMany()
+[]
+
+# Create a new Question.
+⚡ > q = await db.question.create({data: {text: 'What’s new?', publishedAt: new Date()}})
+{ id: 1, text: 'What’s new?', publishedAt: 2020-04-24T22:08:17.307Z }
+
+# Now it has an ID.
+⚡ > q.id
+1
+
+# Access model field values via JavaScript attributes.
+⚡ > q.text
+"What’s new?"
+
+⚡ > q.publishedAt
+2020-04-24T22:08:17.307Z
+
+# Change values by using the update function
+⚡ > q = await db.question.update({where: {id: 1}, data: {text: 'What’s up?'}})
+{ id: 1, text: 'What’s up?', publishedAt: 2020-04-24T22:08:17.307Z }
+
+# db.questions.findMany() displays all the questions in the database.
+⚡ > await db.questions.findMany()
+[
+  { id: 1, text: 'What’s up?', publishedAt: 2020-04-24T22:08:17.307Z }
+]
+```
```

Comment:
Top level await doesn't work right now :(

Reply:
Oh damn:/ It worked for me when I was writing this but maybe that’s a cross-platform thing? Just leaving this here for now since this is the intended behaviour of the console

Reply:
Let's change this to use `.then` so we can go ahead and publish this asap.